core/hs-dtls.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                hs-dtls
-Version:             1.5.0
+Version:             1.6.0
```

Comment:
This package version number matches `hs-tls` version number from which it had been forked. Given that this bump is because of dependencies -- it would be better to name this version 1.5.1.

Reply:
Ok, I do not mind
Just followed my understanding of semantic versioning, where 1.5.1 would mean bugfix, while 1.6.0 is minor change

Reply:
I can see there is a lot of changes, so we can simply keep existing version
https://hackage.haskell.org/package/tls-1.9.0/changelog